drivers/usb/device/usb_dc_stm32.c

```diff
@@ -436,11 +437,17 @@ static int usb_dc_stm32_init(void)
 		k_sem_init(&usb_dc_stm32_state.in_ep_state[i].write_sem, 1, 1);
 	}
 #else /* USB_OTG_FS */
+
 	/* TODO: make this dynamic (depending usage) */
-	HAL_PCDEx_SetRxFiFo(&usb_dc_stm32_state.pcd, FIFO_EP_WORDS);
+	HAL_PCDEx_SetRxFiFo(&usb_dc_stm32_state.pcd, CONFIG_USB_FIFO_RX_MIN);
 	for (i = 0U; i < USB_NUM_BIDIR_ENDPOINTS; i++) {
-		HAL_PCDEx_SetTxFiFo(&usb_dc_stm32_state.pcd, i,
-				    FIFO_EP_WORDS);
+		if (i == 0) {
+			/* first endpoint need only 64 byte for EP_TYPE_CTRL */
+			HAL_PCDEx_SetTxFiFo(&usb_dc_stm32_state.pcd, i,	64);
```

Comment:
You mentioned 64bytes, but the parameter seems to be in word count, not byte.

Reply:
yes you right in fact this value is term of 32 bits : ( see documentation after: )

![image](https://github.com/zephyrproject-rtos/zephyr/assets/119042859/a0a7a577-7105-420f-b264-c148a4709ec8)

Reply:
I push a new modification